Octokit/Models/Request/NewRelease.cs

```diff
@@ -7,7 +7,7 @@ namespace Octokit
     /// Used to create a new release.
     /// </summary>
     /// <remarks>
-    /// API: https://developer.github.com/v3/repos/releases/#create-a-release
+    /// API: https://docs.github.com/en/rest/releases/releases#create-a-release
```

Comment:
Please could you remove the en from the url so GitHub can work out the language appropriately?